### ISSUE TITLE
Kiwix-desktop seems not really compatible with screen readers

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-desktop#739]
Screen readers are used by visually impaired. It looks the famous FOSS screen reader does not work well…

**Comment by kelson42** [comment]
… but I have no clue how to even test this. Any help of someone familiar with this kind of software is welcomed.

**Comment by kaveinthran** [comment]
HI, you can use the free and opensource screenreader nvda, obtained from https://www.nvaccess.org/download/
It's used by many devs to do app and web testing, you can activate speech viewer under tools menu located in the nvda menu to read the words spoken by the screenreader if you have difficulty with the speech.
You can load nvda and run it with kiwix.
another way is to just use the built-in windows Narrator by pressing ctrl+win+enter

happy to help if you need more assistance

**Comment by kelson42** [comment]
@kaveinthran Thx for this first, I shoukd be able to try with a Windows computer... but I'm not sure at all to be able to differenciate the right behaviour from the wrong behaviour. Do you know or would be able to tell me precisely, at least first via a concrete example, an example of what is wrong?

**Comment by Popolechien** [comment]
I see that #570 last year led to a PR and minor improvement, but either that did not work as expected or something else is missing (my guess would be : the actual content, as the fix addresses the tool's UI). 
More info on the concrete use case would be welcome.

**Comment by kelson42** [comment]
@Popolechien Good catch, I had totally forgotten about this ticket. Might simply be that the situation is already a lot better, but because we don't have released since a long time then users don't see it. Pinging @asashnov as he was pretty active on this ticket and therefore the most knowledgeable.

@kaveinthran Have you tried with the latest nightly at https://download.kiwix.org/nightly/ to see if it works better with nvda? Or do you still see problems?

**Comment by kaveinthran** [comment]
HI,

Sorry for the late revert, i've tested both kiwix-desktop_windows_x64_2.0.5 and kiwix-desktop_windows_x64_2022-01-01 with NVDA 2021.3.1 and the results are similar.

1. When I open a zim file, both version do not represent elements, body text to the screen reader. I can't use web navigation or reading keys e.g.e arrows to navigate the document. 
2. the kiwix server method works in the 2022 version but not in the official version. You can feel the difference by using the screen raeder to read in the browser vs kiwix environment.
May I know what design improvements that I've missed to spot in the nightly?

I would love to emphasise here that accessibility is not a feature, it's a design principle.

You can also test kiwix with windows free screen reader, narrator by pressing ctrl+win+enter 
thanks
